### PR TITLE
Release/2.6.0

### PR DESCRIPTION
**Pull request by sven1103**
**Description of changes**

* Add template for OSGi Groovy library bundles
* Add template for OSGi Groovy portlet bundles
* Bump JDK version from 8 to 11
* Fix missing `license` property bug, that showed up if the license placeholder was referenced in a template.

### CODE REVIEW

**Review by jenniferboedker — APPROVED** [review]
LGTM

**Comment by jenniferboedker** [comment]
fixes #211

**Comment by jenniferboedker** [comment]
Please do not merge, I found a small issue when creating the portlet because of a missing dependency. I will commit the required changes

**Review by jenniferboedker — CHANGES_REQUESTED** [review]
The dependency for junit are missing for the java-tests in the osgi-portlet

File: qube/create/templates/cli/cli_java/{{ cookiecutter.project_slug }}/.dependabot/config.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: 1
```
Comment (Zethson) [inline qube/create/templates/cli/cli_java/{{ cookiecutter.project_slug }}/.dependabot/config.yml:1]: Why are you now using the old v1 version again and deleted the new v2 version (.github/dependabot.yml)

File: qube/create/templates/gui/gui_java/{{ cookiecutter.project_slug }}/.dependabot/config.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: 1
```
Comment (Zethson) [inline qube/create/templates/gui/gui_java/{{ cookiecutter.project_slug }}/.dependabot/config.yml:1]: same as above

File: qube/create/templates/cli/cli_java/{{ cookiecutter.project_slug }}/.dependabot/config.yml (hunk 1 above)
Comment (jenniferboedker) [reply qube/create/templates/cli/cli_java/{{ cookiecutter.project_slug }}/.dependabot/config.yml:1]: This is how it was in qOffer 2.0. You suggested me to use the file as in this project

Comment (Zethson) [reply qube/create/templates/cli/cli_java/{{ cookiecutter.project_slug }}/.dependabot/config.yml:1]: Then I must have been wrong with it being correct in qOffer 2.0 or we had a misunderstanding. Sorry for that.

https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates

The file must be stored directly under the .github folder and it should be version 2 and not version 1.

Sorry!

Comment (Zethson) [reply qube/create/templates/cli/cli_java/{{ cookiecutter.project_slug }}/.dependabot/config.yml:1]: https://github.com/qbicsoftware/qoffer-2-portlet/blob/development/.github/dependabot.yml

This is what I mean.
Maybe you were looking at the wrong branch or so.

Comment (sven1103) [reply qube/create/templates/cli/cli_java/{{ cookiecutter.project_slug }}/.dependabot/config.yml]: addressed in 712bc2f

**Review by jenniferboedker — APPROVED** [review]
LGTM

File: qube/create/templates/cli/cli_java/{{ cookiecutter.project_slug }}/.dependabot/config.yml (hunk 1 above)
Comment (Zethson) [reply qube/create/templates/cli/cli_java/{{ cookiecutter.project_slug }}/.dependabot/config.yml]: Dependabot may notify you that the file is supposed to be in a .github folder.

`You must store this file in the .github directory of your repository. `
https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates

But it should still work as long as they support the old paths and versions.

Comment (Zethson) [reply qube/create/templates/cli/cli_java/{{ cookiecutter.project_slug }}/.dependabot/config.yml]: Ahh, @jenniferboedker addressed this already.

Well then :rocket:

**Comment by jenniferboedker** [comment]
The portlet generation now works but still the source folder is not marked automatically

**Pull request merged**